CODE_OF_CONDUCT.md

```diff
@@ -93,6 +93,11 @@ The following behaviours from any participants within the nf-core community (inc
 - Use of sexual or discriminatory imagery, comments, jokes, or unwelcome sexual attention.
 - Verbal and text comments that reinforce social structures of domination related to gender, gender identity and expression, sexual orientation, ability, physical appearance, body size, race, age, religion, or work experience.
 
+Other more subtle behaviours are also against our code of conduct, and may result in an immediate ban:
+
+- Sending unsolicited direct-messages or emails to community members on Slack / other platforms
```

Comment:
```suggestion
- Sending unsolicited direct-messages or emails to community members on Slack / other platforms.
```

Reply:
I think there are a lot of technically unsolicited messages that are totally valid. Like me contacting someone for a bytesize talk. Maybe we need to be careful here...

Reply:
Yeah, guess this is a good point. Maybe emphasize `solely for sale/marketing or similar purposes`?

Reply:
This is a good point.

Reply:
```suggestion
- Sending unsolicited direct-messages or emails solely for marketing purposes to community members.
```

Reply:
Great!